source/include/FreeRTOS_IP_Private.h

```diff
@@ -410,7 +410,7 @@ extern struct xNetworkInterface * pxNetworkInterfaces;
 
 /** @brief Macro calculates the number of elements in an array as a size_t. */
 #ifndef ARRAY_SIZE_X
-    #ifndef _WINDOWS_
+    #if !defined( _WINDOWS_ ) && !defined( __CCRX__ )
```

Comment:
What will happen without this?

Reply:
The `ARRAY_SIZE_X` macro is used in the condition check of for loops, it seems like CC RX doesn't like multiple statements in the condition check part of the for loops (compilation error) similar to windows.